playbooks/templates/dev-install_net_config.yaml.j2

```diff
@@ -7,7 +7,7 @@ network_config:
 - type: ovs_bridge
   name: br-ex
   use_dhcp: true
-  mtu: 1500
+  mtu: {{ network_info.public_ipv4.mtu }}
```

Comment:
you don't technically need this, since if you leave it out, it will be derived from the interfaces in the "members" list. which is 
network_info.public_ipv4.interface 
but there is nothin wrong with explicitely setting it I guess.